backend/config/corpora-api.yml

```diff
@@ -940,6 +940,230 @@ paths:
+                      ethnicity_term_ids:
+                        $ref: "#/components/schemas/wmg_ontology_term_id_list"
+                response_option:
+                  type: string
+                  enum:
+                    - include_filter_dims_include_dataset_links
+                    - include_filter_dims_exclude_dataset_links
+                    - exclude_filter_dims
+              required:
+                - filter
+                - response_option
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              example:
+                {
+                  'expression_summary': {
+                    'gene1': {
+                      'tissuetype1': [
+                        { 'id': 'CL00000', 'me': 0.0, 'n': 0, 'pc': 0.0 },
+                        { 'id': 'CL00001', 'me': 0.0, 'n': 0, 'pc': 0.0 },
+                        { 'id': 'CL00002', 'me': 0.0, 'n': 0, 'pc': 0.0 }
+                      ],
+                      'tissuetype2': [
+                        { 'id': 'CL00000', 'me': 0.0, 'n': 0, 'pc': 0.0 },
+                        { 'id': 'CL00001', 'me': 0.0, 'n': 0, 'pc': 0.0 },
+                        { 'id': 'CL00002', 'me': 0.0, 'n': 0, 'pc': 0.0 }
+                      ]
+                    },
+                    'gene2': {
+                      'tissuetype1': [
+                        { 'id': 'CL00000', 'me': 0.0, 'n': 0, 'pc': 0.0 },
+                        { 'id': 'CL00001', 'me': 0.0, 'n': 0, 'pc': 0.0 },
+                        { 'id': 'CL00002', 'me': 0.0, 'n': 0, 'pc': 0.0 }
+                      ],
+                      'tissuetype2': [
+                        { 'id': 'CL00000', 'me': 0.0, 'n': 0, 'pc': 0.0 },
+                        { 'id': 'CL00001', 'me': 0.0, 'n': 0, 'pc': 0.0 },
+                        { 'id': 'CL00002', 'me': 0.0, 'n': 0, 'pc': 0.0 }
+                      ]
+                    }
+                  },
+                  'term_id_labels': {
+                    'cell_types': [
+                      { 'CL00000': 'CL00000_label' },
+                      { 'CL00001': 'CL00001_label' },
+                      { 'CL00002': 'CL00002_label' }
+                    ],
+                    'genes': [
+                      { 'gene1': 'gene1_label' },
+                      { 'gene2': 'gene2_label' }
+                    ]
+                  }
+                }
+              schema:
+                type: object
+                required:
+                  - expression_summary
+                  - term_id_labels
+                properties:
+                  expression_summary:
+                    type: object
+                    # we use `additionalProperties` instead of `properties`, since the object's property names are
+                    # ontology term ids, rather than a fixed set of names
+                    additionalProperties:
```

Comment:
Looks like connexion does not validate `additionalProperties`, but since I've specified all the detail already, I'll leave it for completeness.